Demos/DemoControllers/TomeDemo.swift

```diff
@@ -0,0 +1,68 @@
+import UIKit
+import FittedSheets
+
+class TomeDemo: UIViewController, Demoable {
```

Comment:
This adds a demo to the demo app. It's the first option.
![Simulator Screen Shot - iPhone 12 - 2021-05-04 at 20 29 22](https://user-images.githubusercontent.com/733352/117052057-8e688680-ad17-11eb-8657-c50d948aae36.png)